app/helpers/miq_task_helper.rb

```diff
@@ -1,4 +1,18 @@
 module MiqTaskHelper
+  include TaskDetailsHelper
+
+  def row_data(label, value)
+    {:cells => {:label => label, :value => value}}
+  end
+
+  def request_task_configuration_script_ids(miq_task)
+    workflow_instance_ids = []
+    if miq_task.context_data.present? && miq_task.context_data[:workflow_instance_id]
+      workflow_instance_ids.push(miq_task.context_data[:workflow_instance_id])
+    end
+    workflow_instance_ids
```

Comment:
I assume the existing component works with >1 configuration_script_id so that's why you need the array?

It would be simpler to do `[miq_task.context_data&.dig(:workflow_instance_id)].compact` if you want to one-line it or
```
workflow_instance_id = miq_task.context_data&.dig(:workflow_instance_id)
return if workflow_instance_id.nil?

[workflow_instance_id]
```

which I think is a little more readable

Reply:
Fixed it, went for the more readable version but can change it if we want the other one.